README.md

```diff
@@ -5,11 +5,18 @@
 ## Getting Started
 
```

Comment:
- [ ] As you guys know that this is a pair-programming project but i cannot see any commit from @Ibnballo1. My humble request is from @Ibnballo1 to make some contributions in this pull request to follow the requirements and submit this pull request again from the dashboard for a review 👍